### PR TITLE
CLOUDP-122971: support interactive commands on e2e tests

### PR DESCRIPTION
**Pull request by gssbzn**
CLOUDP-122971

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

Initial proposal using go-expect to test interactive commands

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,6 +41,7 @@ export PATH := ./bin:$(PATH)
 ifneq ($(OS),Windows_NT)
 	export SHELL := env PATH=$(PATH) /bin/bash
 endif
+export TERM := linux-m
```
Comment (gssbzn) [inline Makefile:44]: to disable colors

**Comment by wtrocki** [comment]
Looks good. 
My only though was that if in future all questions would be defined by some i18n keys it would be easier to reference them in tests.  Then any change in the message/question would not affect the tests.

File: tools/libraryowners/main.go [hunk 2]
```diff
@@ -58,19 +58,40 @@ func newLibraryOwners() (map[string]string, error) {
...
 		if library.Indirect {
 			continue
 		}
-
-		if val, ok := libraryOwner[library.Mod.Path]; !ok {
-			return fmt.Errorf("'%s' is not inside '%s'. Please, remove this dependency from '%s'", library.Mod.Path, libraryOwnersPath, goModpath)
-		} else if val == "" {
-			return fmt.Errorf("'%s' doesn't have a owner. Please, add a owner to %s in '%s", library.Mod.Path, library.Mod.Path, libraryOwnersPath)
+		if val, ok := libOwners[library.Mod.Path]; !ok || val == "" {
+			addedLibs = append(addedLibs, library.Mod.Path)
```
Comment (gssbzn) [inline tools/libraryowners/main.go:68]: @andreaangiolillo FYI I added 3 libs and this was only giving me an error on first find so improving errors here to report all new libs

File: tools/libraryowners/main.go [hunk 3]
```diff
@@ -58,19 +58,40 @@ func newLibraryOwners() (map[string]string, error) {
...
+			}
+		}
+		if !found {
+			removedLibs = append(removedLibs, library)
+		}
+		found = false
+	}
+	if len(removedLibs) != 0 {
+		return fmt.Errorf("%q are not defined, please remove from %q", removedLibs, libraryOwnersPath)
+	}
```
Comment (gssbzn) [inline tools/libraryowners/main.go:94]: added a check when things in the lib owners but not in the go mod

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**